.docker/release/files/entrypoint.sh

```diff
@@ -27,42 +28,47 @@ if [ "${SECURE_COOKIES}" = true ]; then
 fi
 
 # Copy snappymail default config if absent
-SNAPPYMAIL_CONFIG_FILE=/snappymail/data/_data_/_default_/configs/application.ini
+SNAPPYMAIL_CONFIG_FILE=/var/lib/snappymail/_data_/_default_/configs/application.ini
 if [ ! -f "$SNAPPYMAIL_CONFIG_FILE" ]; then
     echo "[INFO] Creating default Snappymail configuration"
     mkdir -p $(dirname $SNAPPYMAIL_CONFIG_FILE)
     cp /usr/local/include/application.ini $SNAPPYMAIL_CONFIG_FILE
 fi
 
 # Enable output of snappymail logs
-if [ "${LOG_TO_STDERR}" = true ]; then
-  sed -z 's/\; Enable logging\nenable = Off/\; Enable logging\nenable = On/' -i $SNAPPYMAIL_CONFIG_FILE
+if [ "${LOG_TO_STDERR}" = 'true' ]; then
+  sed '/^\; Enable logging/{
+N
+s/enable = Off/enable = On/
+}' -i $SNAPPYMAIL_CONFIG_FILE
   sed 's/^filename = .*/filename = "errors.log"/' -i $SNAPPYMAIL_CONFIG_FILE
   sed 's/^write_on_error_only = .*/write_on_error_only = Off/' -i $SNAPPYMAIL_CONFIG_FILE
   sed 's/^write_on_php_error_only = .*/write_on_php_error_only = On/' -i $SNAPPYMAIL_CONFIG_FILE
 else
-    sed -z 's/\; Enable logging\nenable = On/\; Enable logging\nenable = Off/' -i $SNAPPYMAIL_CONFIG_FILE
+  sed '/^\; Enable logging$/{
+N
+s/enable = On/enable = Off/
+}' -i $SNAPPYMAIL_CONFIG_FILE
 fi
 # Always enable snappymail Auth logging
 sed 's/^auth_logging = .*/auth_logging = On/' -i $SNAPPYMAIL_CONFIG_FILE
 sed 's/^auth_logging_filename = .*/auth_logging_filename = "auth.log"/' -i $SNAPPYMAIL_CONFIG_FILE
 sed 's/^auth_logging_format = .*/auth_logging_format = "[{date:Y-m-d H:i:s}] Auth failed: ip={request:ip} user={imap:login} host={imap:host} port={imap:port}"/' -i $SNAPPYMAIL_CONFIG_FILE
+sed 's/^auth_syslog = .*/auth_syslog = Off/' -i $SNAPPYMAIL_CONFIG_FILE
 # Redirect snappymail logs to stderr /stdout
-mkdir -p /snappymail/data/_data_/_default_/logs/
-# empty logs
-cp /dev/null /snappymail/data/_data_/_default_/logs/errors.log
-cp /dev/null /snappymail/data/_data_/_default_/logs/auth.log
-chown -R php-cli:php-cli /snappymail/data/
+mkdir -p /var/lib/snappymail/_data_/_default_/logs/
+ln -sf /dev/stderr /var/lib/snappymail/_data_/_default_/logs/errors.log
```

Comment:
here's how it looks now:

```sh
$ docker exec -it snappymail_snappymail_1 bash
a176e531eac2:/snappymail# ls -al /var/lib/snappymail/_data_/_default_/logs
total 12
drwxr-xr-x    2 www-data www-data      4096 Feb 19 03:27 .
drwxr-xr-x    8 www-data www-data      4096 Feb 17 04:39 ..
-rw-r--r--    1 www-data www-data       218 Feb 19 03:28 auth.log
lrwxrwxrwx    1 www-data www-data        11 Feb 19 03:27 errors.log -> /dev/stderr
```

Reply:
the user should be careful not to mutate the `errors.log` value.

Reply:
Just tested, `errors.log -> /dev/stderr` is not working:

```sh
snappymail_1  | [19-Feb-2023 05:11:27] WARNING: [pool default] child 73 said into stderr: "NOTICE: PHP message: PHP Warning:  error_log(/var/lib/snappymail/_data_/_default_/logs/errors.log): Failed to open stream: No such file or directory in /snappymail/snappymail/v/2.26.1/app/libraries/MailSo/Log/Drivers/File.php on line 53"
```

`php-fpm` workers cannot log to `/dev/stderr` directly, because their `/dev/stderr` redirects to `/dev/null` (see [here](https://dev.to/setevoy/linux-php-fpm-docker-stdout-and-stderr-no-an-application-s-error-logs-4j8d)). Instead, workers must log to stderr directly by using [`error_log()` without a third argument](https://www.php.net/manual/en/function.error-log.php), and their logs will be caught by `php-fpm` master process ( `catch_workers_output = yes` in `php-fpm.conf`) which then will log to `/proc/self/fd/2` for the logs work correctly.

This is fixed in [cee4627](https://github.com/the-djmaze/snappymail/pull/965/commits/cee4627cfbf0bb2161a15d5ab12365bafe365522)